modules/tensor_mechanics/doc/content/modules/tensor_mechanics/CriticalTimeStepMath.md

```diff
@@ -0,0 +1,82 @@
+# Critical Time Step
+
+Explicit time integration schemes are simple to implement, and they require minimal CPU
+time and memory per time step. However, explicit schemes are only conditionally stable.
+That is, the time step used for integration should be small than the so called
+__critical time step__ so the solution remains valid.
```

Comment:
See https://mooseframework.inl.gov/python/MooseDocs/extensions/core.html for underline, bold, etc. syntax